frontend/src/hooks/use-handle-click-outside/use-handle-click-outside.hook.tsx

```diff
@@ -2,24 +2,24 @@ import { RefObject, useEffect } from 'react';
 
 type UseHandleClickOutsideArgs = {
   ref: RefObject<HTMLElement>;
-  onClick: () => void;
+  onClick: (e: React.MouseEvent) => void;
 };
 
 const useHandleClickOutside = ({
   ref,
   onClick,
 }: UseHandleClickOutsideArgs): void => {
   useEffect(() => {
-    const handleClickOutside = (e: MouseEvent): void => {
+    const handleClickOutside = (e: React.MouseEvent | MouseEvent): void => {
```

Comment:
```suggestion
    const handleClickOutside = (evt: React.MouseEvent): void => {
```

Reply:
if i have only `React.MouseEvent` then i get `No overload matches this call.` on `document.addEventListener('click', handleClickOutside)`